rule_book/04_quiz_events.md

```diff
@@ -142,9 +142,7 @@ A foul is a penalty called by a quiz official against a quizzer or team for cond
 - Fouls may be called by any quiz official actively involved in the current quiz.
 - A foul must be called before the quizmaster makes known the next question's type.
 
-The quizzer upon whom a foul is called becomes ineligible to jump on that question number, including any alphanumeric question numbers. The chair remains ineligible to jump on that question number, including any alphanumeric question numbers, even if the quizzer who committed the foul is substituted for.
-
-If the officials fail to call a foul, a coach or captain may call it to the attention of the quizmaster. If in the officials' opinions a foul might have been committed, the quizmaster will give due warning to the violating team. A foul cannot be charged, and points cannot be deducted when the quizmaster receives such help from a coach or captain.
+The quizzer upon whom a foul is called becomes ineligible to jump on that question number.
```

Comment:
Since the sentence before this one ends with "the next question's type," someone might interpret "that question number" to refer to the next question number. Would it be worth adding a phrase to clarify that the quizzer is ineligible to jump on the question number where he receives the foul, rather than the following question number?